pkg/common/si_helper.go

```diff
@@ -42,7 +42,7 @@ func CreateTagsForTask(pod *v1.Pod) map[string]string {
 			if value.Kind == constants.DaemonSetType {
 				if pod.Spec.Affinity == nil || pod.Spec.Affinity.NodeAffinity == nil ||
 					pod.Spec.Affinity.NodeAffinity.RequiredDuringSchedulingIgnoredDuringExecution == nil {
-					log.Logger().Debug("DaemonSet pod's Affinity, NodeAffinity, RequiredDuringSchedulingIgnoredDuringExecution might empty")
+					log.Log(log.ShimContext).Debug("DaemonSet pod's Affinity, NodeAffinity, RequiredDuringSchedulingIgnoredDuringExecution might empty")
```

Comment:
Let's use `log.ShimUtils` for this as that's logically where this goes (helper is a synonym for utility).